conftest.py

```diff
@@ -1,28 +1,45 @@
 """Setup for Pytest."""
 
 # Standard library imports
+from pathlib import Path
+import shutil
+from sys import platform
 import warnings
 import webbrowser
 
 # Third party imports
 import pytest
-
+from PIL import Image, ImageChops
 
 # ---- Constants
 
 OPEN_BROWSER_OPTION = '--open-browser'
+COMPARE_SCREENSHOTS_OPTION = "--compare-screenshots"
+UPDATE_REFERENCE_SCREENSHOTS_OPTION = "--update-reference-screenshots"
 
 
 # ---- Pytest hooks
 
 def pytest_addoption(parser):
-    """Add an option to open the user's web browser with HTML test output."""
+    """Add command line options."""
     parser.addoption(
         OPEN_BROWSER_OPTION,
         action='store_true',
         default=False,
         help='For tests that generate HTML output, open it in a web browser',
     )
+    parser.addoption(
+        COMPARE_SCREENSHOTS_OPTION,
+        action='store_true',
+        default=False,
+        help='For tests that generate HTML output, run visual regression tests on them',
```

Comment:
```suggestion
        help=('For tests that generate HTML output, '
              'run visual regression tests on it'),
```

Plural agreement and 79 chars